cli/io/cbt/cbt.py

```diff
@@ -0,0 +1,189 @@
+import copy
+import json
+import re
+from pathlib import Path
+
+import requests
+import yaml
+
+from cli import Cli
+from cli.utilities.utils import os_major_version
+from utility.log import Log
+from utility.utils import get_ceph_version_from_cluster
+
+log = Log(__name__)
+
+
+class CbtError(Exception):
+    pass
+
+
+class CBT(Cli):
+    def __init__(self, primary_client):
+        super(CBT, self).__init__(primary_client)
+        self.primary_client = primary_client
+        self.server = "http://xx.xx.xx.xx"
```

Comment:
IMHO, we should initialize with actual values as it is in `__init__` function.

Reply:
I couldn't find the setter method but do see this being used in `endpoint_url`. Please add doc strings so that we could understand the purpose of these variables.